### PR TITLE
Fix issue of return status of delete and restore

### PR DESCRIPTION
**Pull request by roboticsexpert**
Problem was on return parent function

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "zizaco/entrust",
+    "name": "roboticsexpert/entrust",
```
Comment (milosdakic) [inline composer.json]: Why the change of package names?

**Comment by milosdakic** [comment]
Does this fix #742?

**Comment by roboticsexpert** [comment]
let me test that !!! @milosdakic

**Comment by roboticsexpert** [comment]
that issue really not bug ... any time you use some trait that have conflict in implementation of same method , you should override that function manually ... @milosdakic

**Comment by Zizaco** [comment]
https://github.com/Zizaco/entrust/commit/5a83c57f0f3a5f7490f6ef76c88bc166b00fdc06
